MangoAPI.Domain/Entities/UserEntity.cs

```diff
@@ -13,8 +13,8 @@ public sealed class UserEntity : IdentityUser<Guid>
 
     public string Bio { get; set; }
 
-    public Guid EmailCode { get; set; }
```

Comment:
Have you updated migrations after email code removed?

If no -- please squash migrations to have only one in PR

Reply:
Yes, I updated migrations